scripts/superdesk-templates/templates.js

```diff
@@ -151,8 +151,6 @@
                 criteria.template_type = type;
             }
 
-            criteria.schedule_desk = null;
```

Comment:
Original thinking was not to show 'schedule' templates. It should not cause any problems if we enable it. 
Please enable it as it is also a "template" but it would be good to have indicator in available templates list that it is a scheduled template as well.